src/test/regress/expected/locally_execute_intermediate_results.out

```diff
@@ -342,69 +341,63 @@ NOTICE:  executing the command locally: WITH a AS (SELECT max(ref_table_1.key) A
 -- some edge cases around CTEs used inside other CTEs
 -- everything can be executed locally
 WITH cte_1 as (SELECT * FROM table_1),
```

Comment:
I guess it'd be nicer if we can change `cte_1` to `MATERIALIZED` as well (here and in the following tests).